README.md

```diff
@@ -15,24 +20,90 @@ yarn
 
 ## Getting started
 
-You can use the demo site to test out changes to the theme. To run the demo
-site, use the following command:
+You can use the `Demo Site` included to test out changes to the theme. It's a much smaller package to build and serve than our other sites, so it's ideal for testing out changes to plugin behaviour, styling and our component library.
+
+To run the `Demo Site`, simply use either of the following commands:
+
+```sh
+yarn start
+```
+
+...or...
 
 ```sh
 yarn workspace demo develop
 ```
 
-Visit [http://localhost:8001](http://localhost:8001) in your browser to browse
-the demo site.
+It should only take `~30 seconds` to build and you should see...
+
+```sh
+success Building development bundle
+```
+
+...in the terminal, then you can visit [http://localhost:8001](http://localhost:8001) in your browser :)
+
+## Editing Content
+
+We use MDX files for content which lets you seamlessly write JSX in your Markdown documents. This means you can write content in regular Markdown format as well as including React componenets in the same file. Gatsby will then do all the heavy-lifting to convert these to HTML to serve in the browser.
+
+### Example
+
+For this example we will show how to add some content to a page and see the render results in the browser. We will use the `build-hello-world-app` document for testing, so with the **develop** server still running, navigate to and open the following in your favourite code editor:
+
+ - `demo/src/content/build-apps/build-hello-world-app.mdx`
+
+You should see something like the following (possibly less wine coloured):
+
+<img src="packages/gatsby-theme-newrelic/src/images/example-mdx.png" alt="drawing" width="75%"/>
```

Comment:
Since these images are for the README, could you move them to a top-level folder in this repo instead of the theme package itself? The `src` directory is packaged with the theme in NPM, so we want to prevent those images from being downloaded as well (which is just wasted space).

Reply:
Good to know, of course will do :D

Reply:
done @jerelmiller :D

Reply:
Thank you!!